tests/integration_tests/performance/test_boottime.py

```diff
@@ -16,46 +16,117 @@
 # Regex for obtaining boot time from some string.
 TIMESTAMP_LOG_REGEX = r'Guest-boot-time\s+\=\s+(\d+)\s+us'
 
+NO_OF_MICROVMS = 10
 
-@pytest.mark.timeout(120)
-def test_microvm_boottime_no_network(test_microvm_with_boottime):
+
+def test_single_microvm_boottime_no_network(test_microvm_with_boottime):
```

Comment:
Why is the 120 s timeout gone?

Reply:
How are we actually enforcing the boot time without the custom timeout? I see we are only printing it and not doing any asserts. Is there another mechanism to enforce that the boot time is less than 120 ms or is this test not suppose to check that?

Reply:
Nvm, I saw that we test it with _test_microvm_boottime

Reply:
@dianpopa I think (might be wrong though) that 300 was chosen with the exact opposite reasoning since we have a heterogeneous mix of short and long duration tests: hard limit the maximum accepted duration, tweak here and there for tests that finish faster.

Reply:
@aghecenco I agree with you but that is why I think  it would be nice if that happened in a consistent way. However, from looking at all the tests from the CI i got the feeling that this customized timeout happens with a certain degree of randomness. If you take a look at `https://github.com/firecracker-microvm/firecracker/blob/master/tests/integration_tests/performance/test_boottime.py` you can see that the `test_microvm_boottime_with_network` does not have it while the `test_microvm_boottime_no_network` has it. 
Moreover, what criteria are we using for deciding that some tests are faster than others? In this specific test we are actually opening a file/FIFO and look through it for the boot time that the machine logged. We are doing IO but we decided a timeout of 120s. For me it still does not make any sense.
If you still want me to add it back, let me know and I will do it.